services/frontend-v3/src/components/profileForms/primaryInfoForm/PrimaryInfoFormView.jsx

```diff
@@ -699,7 +699,7 @@ const PrimaryInfoFormView = ({
                       <Popover
                         content={
                           <div style={styles.popoverStyle}>
-                            <FormattedMessage id="profile.career.header.tooltip" />
+                            <FormattedMessage id="profile.Job.title.tooltip" />
```

Comment:
This PR looks good, I would just make all the letters lowercase in the message id

```suggestion
                            <FormattedMessage id="profile.job.title.tooltip" />
```